### ISSUE TITLE
Crafting Recipes Not Rendering 

### ISSUE THREAD

**Issue by JacquesJnr** [QuantumXenon/origins-plus-plus#293]
*Loader*
Forge

*Versions*
Minecraft Version: 1.19.2
Mod Loader Version:  Forge 0.237.1.14012
- Origins Version:  1.19.2-1.7.1.3
- Origins++ Version: origins-plus-plus-2.2
- Pehkui Version: 3.7.11

*Origin*
None specific - relevant to entire mod.

*Power*
Not origin specific.

*Notes*
**Disclaimer - I run Origins++ alongside a modpack; Valhesia 5 so the complexity maybe beyond the scope for this mod. So if there's no way of recreating the bug, let me know and I'll see about changing versions or only running origins etc.**

My issue is best described by this one posted Mar 22 on the main origins issue page:
https://github.com/EdwinMindcraft/origins-architectury/issues/117

Simply put, when origins++ is active, there's a graphical issue in crafting menus where a recipe does not display it's crafted item.
Is there any way that origins ++ is interfering with GUI outside of the origin selection menu that could be causing this? Either way this bug is only possible with BOTH origins & origins++ enabled, but no issue is present with ONLY origins, hence why I'm asking here.

Has anyone else experienced this? 
Jacques

[![Image from Gyazo](https://i.gyazo.com/7f04b353fe1ed5b97533a668e7b99cab.gif)](https://gyazo.com/7f04b353fe1ed5b97533a668e7b99cab)
